README.md

```diff
@@ -755,6 +755,35 @@ You can also see the help for the command in the CLI:
 $ affinidi verify-vc --help
 ```
 
+### **affinidi rename project**
+
+Change the name of an existing project
```

Comment:
Let's be a bit more user friendly and write something like:
"Use this command to change the name of an existing project"